Document/0x06b-Basic-Security-Testing.md

```diff
@@ -786,42 +757,298 @@ Possible values for the property [UIDeviceFamily](https://developer.apple.com/li
+  - The app can disable paths by setting `NSURLIsExcludedFromBackupKey`.
+- **Library/Preferences/**
+  - Used for storing properties that can persist even after an application is restarted.
+  - Information is saved, unencrypted, inside the application sandbox in a plist file called [BUNDLE_ID].plist.
+  - All the key/value pairs stored using `NSUserDefaults` can be found in this file.
+- **tmp/**
+  - Use this directory to write temporary files that do not need to persist between app launches.
+  - Contains non-persistent cached files.
+  - Invisible to users.
+  - Content in this directory is not backed up.
+  - The OS may delete this directory's files automatically when the app is not running and storage space is running low.
+
+Let's take a closer look at iGoat-Swift's Application Bundle (.app) directory inside the Bundle directory (`/var/containers/Bundle/Application/3ADAF47D-A734-49FA-B274-FBCA66589E67/iGoat-Swift.app`):
+
+```shell
+OWASP.iGoat-Swift on (iPhone: 11.1.2) [usb] # ls
+NSFileType      Perms  NSFileProtection    ...  Name
+------------  -------  ------------------  ...  --------------------------------------
+Regular           420  None                ...  rutger.html
+Regular           420  None                ...  mansi.html
+Regular           420  None                ...  splash.html
+Regular           420  None                ...  about.html
+
+Regular           420  None                ...  LICENSE.txt
+Regular           420  None                ...  Sentinel.txt
+Regular           420  None                ...  README.txt
+
+Directory         493  None                ...  URLSchemeAttackExerciseVC.nib
+Directory         493  None                ...  CutAndPasteExerciseVC.nib
+Directory         493  None                ...  RandomKeyGenerationExerciseVC.nib
+Directory         493  None                ...  KeychainExerciseVC.nib
+Directory         493  None                ...  CoreData.momd
+Regular           420  None                ...  archived-expanded-entitlements.xcent
+Directory         493  None                ...  SVProgressHUD.bundle
+
+Directory         493  None                ...  Base.lproj
+Regular           420  None                ...  Assets.car
+Regular           420  None                ...  PkgInfo
+Directory         493  None                ...  _CodeSignature
+Regular           420  None                ...  AppIcon60x60@3x.png
+
+Directory         493  None                ...  Frameworks
+
+Regular           420  None                ...  embedded.mobileprovision
+
+Regular           420  None                ...  Credentials.plist
+Regular           420  None                ...  Assets.plist
+Regular           420  None                ...  Info.plist
+
+Regular           493  None                ...  iGoat-Swift
+```
+
+Refer to the "Testing Data Storage" chapter for more information and best practices on securely storing sensitive data.
+
+##### Monitoring System Logs
+
+MYTODO create.
```

Comment:
dont' forget about it 👍

Reply:
done, I've moved existing content from another section and included passionfruit